app/views/posts/index.html.erb

```diff
@@ -1,3 +1,12 @@
+<link href="https://fonts.googleapis.com/css?family=Tenor+Sans&display=swap" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="base.css">
+
+<div class="topnav">
+  <a class="active" href="#home">Home</a>
+  <a href="#create">Create</a>
+<a href="#friends">Friends</a>
```

Comment:
We'll want to update these when we add more routes but I think they're OK as placeholders for now